sections/semantics-sections.include

```diff
@@ -1101,7 +1101,7 @@
     </xmp>
 
     The following snippet shows how one might use headers within a blog. The first header
-    acting as introductory content for the page, as a whole, scoped to the <{body}>
+    acting as introductory content for the page as a whole, and is scoped to the <{body}>
```

Comment:
s/acting as/provides/ or something. (pedantry: so there is an explicit finite verb in the principle clause...)